.gitignore

```diff
@@ -16,3 +16,6 @@
 /test/src/*.dylib
 /test/src/*.o
 /test/src/*.so
+
+# Do NOT ignore binaries placed in subdirs under '/test/bin'.
+!/test/bin/*/*
```

Comment:
Why is this needed? I can't seem to find the rule(s) that would cause the files that are un-ignored here to be ignored in the first place.